packages/daimo-api/src/contract/foreignCoinIndexer.ts

```diff
@@ -152,12 +166,23 @@ export class ForeignCoinIndexer extends Indexer {
       );
 
       // Special case: we previously swapped ETH in bulk
-      // If there's an unmatched outbound, clear prior inbounds.
+      // If there's an unmatched outbound, replace all prior inbounds with a single one for the full balance.
```

Comment:
this ensures we don't have dangling, unclaimable token balances in people's V1 accounts